firefox_ui_tests/functional/locationbar/test_escape_autocomplete.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestEscapeAutocomplete(FirefoxTestCase):
+
+    def tearDown(self):
+        autocompleteresults = self.browser.navbar.locationbar.autocomplete_results
+
+        if autocompleteresults.is_open:
+            autocompleteresults.close(force=True)
+            self.wait_for_condition(not autocompleteresults.is_open)
```

Comment:
There is no need to check if the auto-complete is open. Just call `close()` with force set to True. Also this method itself checks if it is closed. So all those 5 lines above can become a single one.